hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/CompactSplit.java

```diff
@@ -218,14 +221,14 @@ public synchronized boolean requestSplit(final Region r) {
     return false;
   }
 
-  public synchronized void requestSplit(final Region r, byte[] midKey) {
+  private synchronized void requestSplit(final Region r, byte[] midKey) {
     requestSplit(r, midKey, null);
   }
 
   /*
    * The User parameter allows the split thread to assume the correct user identity
    */
-  public synchronized void requestSplit(final Region r, byte[] midKey, User user) {
+  private synchronized void requestSplit(final Region r, byte[] midKey, User user) {
```

Comment:
What is the reason to change the visibility of this method?

Reply:
As above, they are additional minor modifications.